README.md

```diff
@@ -2,74 +2,50 @@
 
 # Project Name
```

Comment:
- [ ] [OPTIONAL] you did a great job using the microverse readme template 👏👏 Kindly Change `Project Name` to your actual project name to make the readme more descriptive.